packages/appkit/package.json

```diff
@@ -61,6 +61,11 @@
       "types": "./dist/types/exports/auth-provider.d.ts",
       "import": "./dist/esm/exports/auth-provider.js",
       "default": "./dist/esm/exports/auth-provider.js"
+    },
+    "./adapters": {
+      "types": "./dist/types/exports/adapters.d.ts",
+      "import": "./dist/esm/exports/adapters.js",
+      "default": "./dist/esm/exports/adapters.js"
```

Comment:
is this ok? won't this duplicate the adapters ? They are currently exposed as `@reown/appkit-adapter-whatever`

Reply:
We need this to import the blueprint class